examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -69,6 +70,8 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
             }
         }
 
+        viewModel.retrieveCustomerInfo()
```

Comment:
Should we remove the previous `getCustomerInfoWith` call now that this is here?

Reply:
Fixed in [aebd8ce](https://github.com/RevenueCat/purchases-android/pull/1012/commits/aebd8ce10f16e629f423041f83e25156412271ab)